### PR TITLE
Do not include unbootable disks in bootloader list.

### PR DESCRIPTION
**Pull request by dwlehman**
This is untested, but I think something along these lines makes sense to prevent SAN disks from being selected as the boot device.

### CODE REVIEW

**Comment by jkonecny12** [comment]
Hi Dave could you please rebase this to `master` branch.

File: pyanaconda/kickstart.py [hunk 1]
```diff
@@ -557,7 +557,8 @@ def _is_usable_disk(self, d):
         return \
             not d.format.hidden and \
             not d.protected and \
-            not (blivet.arch.is_s390() and isinstance(d, iScsiDiskDevice))
+            not (blivet.arch.is_s390() and isinstance(d, iScsiDiskDevice)) and \
```
Comment (M4rtinK) [inline pyanaconda/kickstart.py]: I think there should be a comment about what this code does:

`('local' in d.tags or getattr(d, 'ibft', False))`

And ideally also about the other conditions on the line (the s390 & iScsiDiskDevice) are there and preclude the device from being used for for the bootloder.

Comment (jkonecny12) [reply pyanaconda/kickstart.py]: Or move this to a separate method.

**Comment by jkonecny12** [comment]
Could you please explain why do you think this is happening? I'm not sure I understand this change set.

**Comment by dwlehman** [comment]
Sure! If there is no explicit bootloader ordering given, we select the "first" bootable disk. We cannot boot from iSCSI disks unless they are configured using iBFT. We do filter out iSCSI disks on s390, but we don't have anything to rule out non-iBFT iSCSI disks in general. That's what this adds.

**Comment by jkonecny12** [comment]
Thanks for explanation, finally I understand now.
Could you please add this to the commit message.

**Comment by poncovka** [comment]
Hi, it looks like the problem was somewhere else. Anaconda was able to choose a valid disk, but it didn't specify it for the allocation of partitions in Blivet and Blivet chose an invalid one. See: https://github.com/rhinstaller/anaconda/pull/2680

I am closing this PR.

**Pull request closed without merging**